### DISCUSSION TITLE
Spectated player doesn't see who is spectating them

### DISCUSSION THREAD

**Discussion by poettig** [ppy/osu#21929] [Ideas] [answered]
In Stable, it is shown in the upper left corner who is currently spectating you. In Lazer, this is missing (or I am overlooking something). 

Is this on purpose or just "not implemented yet"? If the former, what is the reasoning? 

IMO this would be a good "default on, but can be disabled in options" feature.

**Comment by frenzibyte** [comment, marked as answer]
This is not implemented yet.

**Reply by poettig** [reply]
I see, thanks!

**Reply by peppy** [reply]
https://github.com/ppy/osu/issues/22087
